package.json

```diff
@@ -24,44 +24,54 @@
   },
   "dependencies": {
     "classnames": "*",
+    "create-react-class": "^15.6.3",
     "deepmerge": "^1.4.0",
     "lodash.debounce": "4.0.3",
     "lodash.values": "4.1.0",
-    "react": "0.14.7",
-    "react-dom": "0.14.7",
-    "react-scrollbar": "0.4.0",
+    "prop-types": "^15.6.1",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
+    "react-scrollbar": "^0.4.0",
     "screenfull": "3.0.0"
   },
   "devDependencies": {
-    "browserify": "13.0.0",
+    "@babel/core": "^7.0.0-beta.51",
+    "@babel/preset-env": "^7.0.0-beta.51",
+    "@babel/preset-react": "^7.0.0-beta.51",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^23.2.0",
+    "babel-polyfill": "^6.26.0",
+    "babelify": "^9.0.0",
+    "browserify": "^16.2.2",
     "bulk-require": "^1.0.0",
-    "bulkify": "^1.4.2",
+    "bulkify": "^1.4.0",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
     "eslint": "2.13.1",
     "eslint-plugin-react": "3.7.0",
     "git-rev": "0.2.1",
     "graceful-fs": "*",
-    "gulp": "3.9.1",
-    "gulp-connect": "^2.3.1",
-    "gulp-open": "^1.0.0",
-    "gulp-rename": "*",
-    "gulp-replace": "0.5.4",
-    "gulp-sass": "2.0.4",
-    "gulp-shell": "0.4.2",
-    "gulp-sourcemaps": "1.5.2",
-    "gulp-uglify": "1.1.0",
-    "gulp-util": "3.0.7",
-    "jest-cli": "0.4.18",
-    "jquery": "1.8.3",
+    "gulp": "^3.9.1",
+    "gulp-connect": "^5.5.0",
+    "gulp-open": "^3.0.1",
+    "gulp-rename": "^1.3.0",
+    "gulp-replace": "^1.0.0",
+    "gulp-sass": "^4.0.1",
+    "gulp-shell": "^0.6.5",
+    "gulp-sourcemaps": "^2.6.4",
+    "gulp-uglify": "^3.0.0",
+    "gulp-util": "^3.0.8",
+    "jest": "^23.1.0",
+    "jest-cli": "^23.1.0",
+    "jquery": "^3.3.1",
```

Comment:
Not 100% sure, but I think the skin actually gets the jquery reference from the core. If that's the case then it might make sense to keep the old version in order to test with the one that will actually be loaded by the player.

Reply:
Yep, you're correct. I'll change this back.

Reply:
Node is throwing errors when attempting to install our older version (1.8.3). I'll look into this a little to see if it is easy to resolve this. However, it may not be possible.